specs/merkle-tree.md

```diff
@@ -147,6 +145,8 @@ Outputs a [SphinxMerkleTree](https://github.com/sphinx-labs/sphinx/blob/develop/
 ## Dependencies
 The Merkle tree generation utility makes calls to external libraries. We use `ethers` version 6.7.0 to handle ABI encoding tree leaf `data` and `@openzeppelin/merkle-tree` version 1.0.5 to assemble the Merkle tree. We test that interactions with these libraries work correctly in our [SphinxModuleProxy tests](https://github.com/sphinx-labs/sphinx/blob/develop/packages/contracts/test/SphinxModuleProxy.t.sol) which test that Merkle trees generated by this logic have data that is encoded correctly and that they are executable on-chain. However, we do not test the internals of these libraries and instead rely on the assumption that they are robust and bug-free.
```

Comment:
I know this is out of scope for the PR, but it'd be nice to link to the version of `@openzeppelin/merkle-tree` that we use so readers doesn't need to dig it up themselves